tests/ga/test_extension.py

```diff
@@ -2503,20 +2497,19 @@ def _set_dependency_levels(self, dependency_levels, exthandlers_handler):
         all_handlers = []
         for handler_name, level in dependency_levels:
             if handler_map.get(handler_name) is None:
-                handler = ExtHandler(name=handler_name)
-                extension = Extension(name=handler_name)
-                handler.properties.state = ExtHandlerRequestedState.Enabled
-                handler.properties.extensions.append(extension)
+                handler = Extension(name=handler_name)
+                extension = ExtensionSettings(name=handler_name)
+                handler.state = ExtensionRequestedState.Enabled
+                handler.settings.append(extension)
                 handler_map[handler_name] = handler
                 all_handlers.append(handler)
 
             handler = handler_map[handler_name]
-            for ext in handler.properties.extensions:
+            for ext in handler.settings:
                 ext.dependencyLevel = level
 
-        exthandlers_handler.ext_handlers.extHandlers = []
-        for handler in all_handlers:
-            exthandlers_handler.ext_handlers.extHandlers.append(handler)
+        exthandlers_handler.protocol.client.get_extensions_goal_state()._extensions *= 0
```

Comment:
Out of curiosity, what does this do - `*= 0`? Does this initialize a new array with the value 0? Does simply using ` = []` not work?

Reply:
clears the array (python2 does not have a list.clear() method).

i want to clear the same instance, rather than replacing it with a new empty instance